src/sentry/api/event_search.py

```diff
@@ -82,7 +82,7 @@ def translate(pat):
 
 # has filter for not null type checks
 has_filter      = negation? "has" sep (search_key / search_value)
-is_filter       = negation? "is" sep (search_key / search_value)
+is_filter       = negation? "is" sep search_value
```

Comment:
Having these as keys was causing the names to get remapped, which isn't expected behaviour. Since we validate these values on a mapping of valid fields this should be fine regardless